src/test/k6/CodeEditor.js

```diff
@@ -19,14 +19,16 @@ const adminUsername = __ENV.ADMIN_USERNAME;
 const adminPassword = __ENV.ADMIN_PASSWORD;
 let baseUsername = __ENV.BASE_USERNAME;
 let basePassword = __ENV.BASE_PASSWORD;
+// Use users with ID >= 100 to avoid manual testers entering the wrong password too many times interfering with tests
+const userIdOffset = 99;
 
 export function setup() {
     // Create course as admin
     const artemisAdmin = login(adminUsername, adminPassword);
     const courseId = newCourse(artemisAdmin).id;
 
-    const instructorUsername = baseUsername.replace('USERID', '1');
-    const instructorPassword = basePassword.replace('USERID', '1');
+    const instructorUsername = baseUsername.replace('USERID', '101');
+    const instructorPassword = basePassword.replace('USERID', '101');
```

Comment:
Why are you not using the offset here as well?

Reply:
Because i had to set a specific user to instructor on the Testservers. Using the offset just didnt make sense in this case.